### ISSUE TITLE
Data stores with mask characters while form submit. How to unmask and get typed value during form sumbit?

### ISSUE THREAD

**Issue by sathishbabu007** [RobinHerbots/Inputmask#361]
If I have mask like

1(999)(2233)

while form saving its store like 1(999)(2233). I donot want this. I want only type chars.

eg. 19992233

**Comment by RobinHerbots** [comment]
@sathishbabu007 , 

If you set the autounmask to true, it should work on FF and browsers which support overriding native properties. , ...

For now you can hook into the submit event and create your own formdata to submit with the unmaskedvalues.

Or when using ajax.  Add the autounmask option  and get the values by jquery.val()

**Comment by RobinHerbots** [comment]
There is a new option removeMaskOnSUbmit which removes the mask before submitting.  You can also use jQuery.serialize
